src/lib/services/transfer/service/access.ts

```diff
@@ -0,0 +1,134 @@
+import { HTTP_METHODS, serviceRequest } from '../../shared.js';
+
+import { ID, SCOPES } from '../config.js';
+
+import type { ServiceMethodDynamicSegments, JSONFetchResponse } from '../../../services/types.js';
+import { Transfer } from '../types.js';
+
+/**
+ * @see https://docs.globus.org/api/transfer/acl/#rest_access_get_list
+ */
+export const getAll = function (endpoint_xid, options?, sdkOptions?) {
+  return serviceRequest(
+    {
+      service: ID,
+      scope: SCOPES.ALL,
+      path: `/v0.10/endpoint/${endpoint_xid}/access_list`,
+    },
+    options,
+    sdkOptions,
+  );
+} satisfies ServiceMethodDynamicSegments<
+  string,
+  {
+    query?: Transfer['Paging']['Offset']['Query'] & Record<string, string>;
+    headers?: Record<string, string>;
```

Comment:
```suggestion
    query?: Transfer['Paging']['Offset']['Query']
```

You shouldn't need to `& Record<string, string>;` or define `headers?: ...` here. In the `ServiceMethodDynamicSegments` will intersect this object type with those defaults.